README.md

```diff
@@ -84,6 +84,7 @@ These entrypoints are mainly for information purposes and required to build a me
         - `has_consensus` bool that indicates if the proposed tx was confirmed by a sufficient amount of signers
         - `expiration_height` the block height where the proposed tx expires
         - `expired` bool that indicates if the proposed tx is expired
+        - `proposed_by` refers to the address of the individual who initiated the proposal
```

Comment:
bit wastefull to save it in state additionally if not used in contract logic

Reply:
we have that already stored in the state. Badi just added it to the consensus_info response. it is probably debatable but doesn't really hurt IMO

Reply:
right, then its not a problem, my mistake

Reply:
we had to make many requests to retrieve the person, who created the proposal
I think it should come along with the census info